### PR TITLE
complete tests list

### PR DESCRIPTION
**Pull request by yurunhub**

### CODE REVIEW

File: test/debts.js [hunk 1]
```diff
@@ -47,13 +47,62 @@ contract('Debts', function(accounts) {
...
   });
 
-  it('should allow to borrow');
+  it('should allow to borrow', () => {
+    const borrower = accounts[3];
+    const value = 1000;
+    return Promise.resolve()
+    .then(() => debts.borrow(value, {from: borrower}))
+    .then(() => debts.debts(borrower))
+    .then(asserts.equal(1000));
```
Comment (lastperson) [inline test/debts.js]: You already have value const, maybe use it here?

**Review by lastperson — APPROVED** [review]
What should happen on zero borrow or repay?

**Comment by yurunhub** [comment]
>What should happen on zero borrow or repay?

Per my understanding there should not be an Event emitted, however Depts.sol contract would not pass this check so I only added the check in my Lender.sol contract and tests

File: test/lender.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    const borrower = accounts[3];
+    const value = 1000;
+    return Promise.resolve()
+    .then(() => lender.requestMoney(value, {from: borrower}))
+    .then(() => lender.returnMoney(borrower, value, {from: OWNER}))
+    .then(() => lender.balanceOf(borrower))
+    .then(asserts.equal(0));
+  });
+
+  it('should fail on overflow when requesting money', () => {
```
Comment (lastperson) [inline test/lender.js]: is there an overflow or something else?

File: test/lender.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+  it('invented: don\'t emit event when repaying 0', () => {
+    const owner = accounts[0];
+    const borrower = accounts[3];
+    const value = 1000;
+    return Promise.resolve()
+    .then(() => lender.requestMoney(value, {from: borrower}))
+    .then(() => asserts.throws(lender.returnMoney(0, {from: borrower})));
+  });
+
```
Comment (lastperson) [inline test/lender.js]: test for borrowing 50000000 and then another 50000001.

**Review by lastperson — CHANGES_REQUESTED** [review]
All the functions call results should indicate to caller if they went alright or failed. In your case functions with onlyOwner/onlyNotOwner silently fail in modifier, and caller cannot verify if it failed or not.

**Review by lastperson — APPROVED** [review]

**Pull request closed without merging**